stage1/rootfs/usr_from_coreos/install

```diff
@@ -1,3 +1,5 @@
 mkdir -p "$ROOT"
 cp -a ../usr_from_coreos/rootfs/. "$ROOT"
 ln -sf "coreos" "$ROOT/flavor"
+# TODO change this when the coreos image is updated
```

Comment:
Can you add a comment in `stage1/rootfs/usr_from_coreos/cache.sh` before `IMG_RELEASE="444.5.0"`:
```
# When changing the image, update the systemd version in stage1/rootfs/usr_from_coreos/install.
```

Then, we will remember and we would not need this TODO.